src/web/rustdoc.rs

```diff
@@ -336,9 +332,10 @@ impl RustdocPage {
             result => result.context("error rewriting HTML")?,
         };
 
+        let robots = if is_latest_url { "" } else { "noindex" };
```

Comment:
Do search engines accept an empty string here? or should we omit the header altogether?

Reply:
I'm pretty sure search engines will ignore the empty string.

I'd be happy to omit the header altogether but I couldn't quite figure out how to do it tidily with axum's way of turning a tuple into a response.